pkgs/servers/mpd/Revert-lib-yajl-drop-yajl-prefix-from-include-paths.patch

```diff
@@ -0,0 +1,88 @@
+From fd491775de3f457d9d873779471cef1bf83df356 Mon Sep 17 00:00:00 2001
+From: misuzu <bakalolka@gmail.com>
+Date: Sat, 29 Mar 2025 16:36:11 +0200
+Subject: [PATCH] Revert "lib/yajl: drop "yajl/" prefix from #include paths"
+
+This reverts commit 0aeda01ba6d22a8d9fc583faa67ffc6473869a43.
+---
+ src/lib/yajl/Callbacks.hxx | 7 +++++--
+ src/lib/yajl/Gen.hxx       | 7 +++++--
+ src/lib/yajl/Handle.hxx    | 7 +++++--
+ 3 files changed, 15 insertions(+), 6 deletions(-)
+
+diff --git a/src/lib/yajl/Callbacks.hxx b/src/lib/yajl/Callbacks.hxx
+index f2bc9ffe3..73039ebc9 100644
+--- a/src/lib/yajl/Callbacks.hxx
++++ b/src/lib/yajl/Callbacks.hxx
+@@ -1,13 +1,14 @@
+ // SPDX-License-Identifier: BSD-2-Clause
+ // author: Max Kellermann <max.kellermann@gmail.com>
+ 
+-#pragma once
++#ifndef YAJL_CALLBACKS_HXX
++#define YAJL_CALLBACKS_HXX
+ 
+ #include "util/Cast.hxx"
+ 
+ #include <string_view>
+ 
+-#include <yajl_parse.h>
++#include <yajl/yajl_parse.h>
```

Comment:
I think we should be able to replace the patch with a `substituteInPlace` here. That way we can reduce the diff as it's really just a single line change for each file

Reply:
Done!